arenadata/Dockerfile.ubuntu

```
@@ -29,27 +29,42 @@ RUN set -eux; \
     pip2 install allure-behave==2.4.0; \
     pip2 cache purge
 
+RUN set -eux; wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf -; \
```

use one command per line
```suggestion
RUN set -eux; \
    wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf -; \
```